worldengine/draw.py

```diff
@@ -9,6 +10,36 @@
 # Helper values
 # -------------
 
+### For draw_satellite ###
+NOISE_RANGE = 10 # a random value between -NOISE_RANGE and NOISE_RANGE will be added to the rgb of each pixel
+
+# These are arbitrarily-chosen elevation cutoffs for 4 different height levels. 
+# Some color modifiers will be applied at each level
+HIGH_MOUNTAIN_ELEV = 215
```

Comment:
The function I mentioned can be found [here](https://github.com/Mindwerks/worldengine/blob/master/worldengine/simulations/basic.py). You ask it *"What is the absolute value in height when I only want to look at the highest 10% of the map?"* and it'll answer *"229."* in case you provided it with an elevation-map normalized to the range [0,255].
This can be done with any kind of map (on the temperature-map you would look for the hottest spots, etc.). Normally the function is used because the exact range of values available for a single world is not exactly known. Elevations might range from [0,10.5] or maybe [0.1, 16].

I hope that wasn't too convoluted.